nmagent/requests.go

```diff
@@ -67,6 +71,65 @@ type PutNetworkContainerRequest struct {
 	PrimaryAddress string `json:"-"`
 }
 
+type internalNC struct {
+	// NMAgent expects this to be a string, except that the contents of that string have to be a uint64.
+	// Therefore, the type we expose to clients uses a uint64 to guarantee that, but we
+	// convert it to a string here.
+	Version string `json:"version"`
```

Comment:
What has DNC been sending to CNS? A string or a json number?

Reply:
@ramiro-gamarra It's been sending a JSON number. It needs to be a string.

Reply:
DNC has been sending a string